src/react/src/__tests__/components/ClaimFacilityIntroStep.test.js

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+
+import ClaimFacilityIntroStep from '../../components/ClaimFacilityIntroStep';
+import renderWithProviders from '../../util/testUtils/renderWithProviders';
+import { updateClaimFacilityIntro } from '../../actions/claimFacility';
+
+
+describe('ClaimFacilityIntroStep component', () => {
+    it('renders without crashing', () => {
+        renderWithProviders(
+            <ClaimFacilityIntroStep/>
+        )
+    });
+
+    it('select "Yes" radio button by action', () => {
+        const {reduxStore} = renderWithProviders(
+            <ClaimFacilityIntroStep agreement="true" updateAgreement={() => {}}/>
```

Comment:
Use `{true}` instead of string notation.

Reply:
done

Reply:
As I see there is still "true" @Innavin369

Reply:
+1 @roman-stolar

Reply:
It looks like there is misconception here - you store "yes" / "no" to the component state while switching radiobuttons and then store true / false to the Redux state. I suggest to store "yes" / "no" to the Redux state as well.